### DISCUSSION TITLE
How to show members account on github organization?

### DISCUSSION THREAD

**Discussion by denitiawan-ghcr** [community/community#54374] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to show members account on github organization?

**Comment by denitiawan** [comment, marked as answer]
How to show members account on github organization? 
- go to  your organization
- click tab "people"
- select the member you wanna to change
- change private to public

refresh your organization page, and you can see the members account will show on organization page

**Reply by denitiawan-ghcr** [reply]
thank you @denitiawan

**Comment by nirav-gajera** [comment]
In the top right corner of GitHub.com, click your profile photo, then click Your organizations.
Click the name of your organization.
Under your organization name, click People.
Locate your username in the list of members. If the list is large, you can search for your username in the search box.
Next to your username, select the visibility dropdown menu, then click a new visibility- To publicize your membership, choose Public.
To hide your membership, choose Private.
